sdks/java/src/main/proto/agones/sdk.proto

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 Google LLC All Rights Reserved.
```

Comment:
I don't suppose there is anything we can do here to not have a copy of the file here -- I'm concerned with remembering to keep them in sync?

Maybe we can use a symlink or copy it on every gen.sh? (and ignore it otherwise?)

Reply:
I managed to use a symlink. Some changed were needed as the sdk.proto in root isn't java specific. Compiles like a charme now. I also added some missing dependencies, it should be possible to write tests now.